### PR TITLE
Feature/AN-20: Login page working

### PR DESCRIPTION
**Pull request by ArkadiuszMin**
**Jira ticket**
[AN-20] https://agh-student.atlassian.net/jira/software/c/projects/AN/boards/3?selectedIssue=AN-20

### CODE REVIEW

File: frontend/sumo-championship/src/contexts/UserContext.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    signOut: () => void
+}
+
+export const userContext = createContext<UserContextType>({
+    user: defaultUser,
+    signIn: (user: loginInformation) => new Promise((res, rej) => {}),
+    signOut: () => {}
+});
+
+const UserContext = (props: { children: any; }) => {
```
Comment (Skalakid) [inline frontend/sumo-championship/src/contexts/UserContext.tsx:33]: The context looks good, however I found [pretty interesting article](https://kentcdodds.com/blog/how-to-use-react-context-effectively) about how to use contexts in more optimal and efficient way. I used that in my previous project and it was perfect. You can find many templates/examples there

File: frontend/sumo-championship/src/styles/Atoms.css
Comment (Skalakid) [inline frontend/sumo-championship/src/styles/Atoms.css:1]: I'm not too familiar with this file naming technique, why Atoms and Molecules?

**Review by Skalakid — COMMENTED** [review]
LGTM but I left some questions below. Let's wait for backend changes, and then I will approve this PR

**Comment by Skalakid** [comment]
@ArkadiuszMin also please add description with the JIRA ticket to the PR

File: frontend/sumo-championship/src/styles/Atoms.css
Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/styles/Atoms.css:1]: https://szkolareacta.pl/atomic-design/ - basically atomic design

Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/styles/Atoms.css:1]: and if you asking about css file naming - i have no idea how is it done, is it one file per component, one file in general, this just made sense to me, but I'm not css guy :<

File: frontend/sumo-championship/src/contexts/UserContext.tsx (hunk 1 above)
Comment (ArkadiuszMin) [reply frontend/sumo-championship/src/contexts/UserContext.tsx:29]: Nice article, thanks! Took from that custom hook and undefined "trick" in typescript, but I think reducers are much over the top. Not really readable, simply seems like pure redux

**Review by Skalakid — APPROVED** [review]

**Pull request merged**